packages/dashboard/src/components/dashboard/index.tsx

```diff
@@ -24,13 +24,20 @@ export type DashboardProperties = {
   onSave: DashboardSave;
   clientConfiguration: DashboardClientConfiguration;
   dashboardConfiguration: DashboardConfiguration;
+  initialViewMode?: 'preview' | 'edit';
 };
 
-const Dashboard: React.FC<DashboardProperties> = ({ onSave, clientConfiguration, dashboardConfiguration }) => {
+const Dashboard: React.FC<DashboardProperties> = ({
+  onSave,
+  clientConfiguration,
+  dashboardConfiguration,
+  initialViewMode,
+}) => {
+  const readOnly = initialViewMode && initialViewMode === 'preview';
```

Comment:
nit: why check for truthiness of `initialViewMode` when checking for string equality (`initialViewMode === 'preview'`)?